testing/e2e/test_cases/workload_create_test.go

```diff
@@ -0,0 +1,73 @@
+//go:build integration
+// +build integration
+
+/*
+Copyright 2022 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/testing/e2e/helpers"
+	it "github.com/vmware-tanzu/apps-cli-plugin/testing/e2e/suite"
+)
+
+func TestCreateFromGitWithAnnotations(t *testing.T) {
+	testSuite := it.CommandLineIntegrationTestSuite{
+		{
+			Name:         "Create workload with valid name from a git repo and annotations",
+			WorkloadName: "test-create-git-annotations-workload",
+			Command: func() it.CommandLine {
+				c := *it.NewCommandLine(
+					"tanzu", "apps", "workload", "create", "test-create-git-annotations-workload",
```

Comment:
`tanzu apps` will default in all the commands so `NewCommandLine` can always prepend this as part of the function call. Less function args in that case.  If you end up doing this then rename func name to `NewTanzuAppsCommandLine`

Reply:
added the new function `NewTanzuAppsCommandLine`